app/models/ranking_entry.rb

```diff
@@ -0,0 +1,70 @@
+# Smallest unit of a ranking list. Contains different data for one team and contains logic to parse those date out of
```

Comment:
Typo in `date`:
```diff
- … those date out of
+ … those data out of
```

Depending on your preference, you could also change `those` to `these` – see https://www.englisch-hilfen.de/grammar/this_that.htm:

> Ist etwas näher dran, nimmt man this/these, ist etwas weiter weg (zeitlich und örtlich) steht that/those.